stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/decorator/internal/ReplyDecorator.kt

```diff
@@ -8,7 +8,7 @@ import io.getstream.chat.android.ui.message.list.adapter.viewholder.internal.Gip
 import io.getstream.chat.android.ui.message.list.adapter.viewholder.internal.MessagePlainTextViewHolder
 import io.getstream.chat.android.ui.message.list.adapter.viewholder.internal.TextAndAttachmentsViewHolder
 
-internal class ReplyDecorator(private val getCurrentUser: () -> User) : BaseDecorator() {
+internal class ReplyDecorator(private val getCurrentUser: () -> User?) : BaseDecorator() {
```

Comment:
Seems like we should pass there Flow<User> or LiveData<User>. Maybe we can iterate over it and do later, but then we should put new task to our backlog

Reply:
There was no need to make the return optional... that was probably a noise in the code. Removed.